tests/unit/components/sl-date-time-test.js

```diff
@@ -16,15 +16,28 @@ test( 'Default class names are present', function( assert ) {
 
 test( 'Attribute "datetime" is properly set', function( assert ) {
     this.subject({
-        timezone: 'America/Chicago',
-        value: new Date( 2015, 0, 1 )
+        format: 'datetime',
+        timezone: 'America/Denver',
```

Comment:
The real fix to the test is on line 21 (replaces line 20 of original). Instantiating a datetime via new Date ( 2015, 0, 1) is the variable as it uses the browser timezone and comparing the values will only pass if the test is run on a machine with that specified timezone. It won't break in MST.